### PR TITLE
website: add partnerships page

### PR DESCRIPTION
**Pull request by mwickett**
🔍 [Preview](https://nomad-gqom94sd2-hashicorp.vercel.app/docs/partnerships)

Stubs in partnerships page for @changli0617 to fill with content.

### CODE REVIEW

**Comment by changli0617** [comment]
@mwickett Please review. Thanks.

**Review by changli0617 — COMMENTED** [review]
update a wrong link

File: website/content/docs/partnerships.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+---
+
+# Nomad Integration Program
+
+
+## Introduction
+
+The HashiCorp Nomad Integration Program allows vendors to integrate their products to work with Nomad. As a scheduler, Nomad encompasses a broad area.  Thereby, a set of integrations may require vendor integration code.
+ 
+Vendors integrating their solutions via the Nomad Integration Process provide their customers a verified and seamless user experience. The Nomad Integration Program primarily supports coding with the Gco programming language.
```
Comment (angrycub) [inline website/content/docs/partnerships.mdx:14]: typo?
```suggestion
Vendors integrating their solutions via the Nomad Integration Process provide their customers a verified and seamless user experience. The Nomad Integration Program primarily supports coding with the Go programming language. 
```

File: website/content/docs/partnerships.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+This information is recorded and used by HashiCorp to track the integration through various stages. The information is also used to notify the integration developer of any overlapping work, perhaps coming from the community so you may better focus resources.
+
+Nomad has a large and active community and ecosystem of partners that may have already started working on a similar integration. We'll do our best to connect similar parties to avoid duplicate work.
+
+### 2. Enable
+
+While not mandatory, HashiCorp encourages vendors to sign and MNDA (Mutual Non-Disclosure Agreement) to allow for open dialog and sharing of ideas during the integration process.
+
+In an effort to support our self-serve model we’ve included links to resources, documentation, examples and best practices to guide you through the Vault integration development and testing process.
```
Comment (angrycub) [inline website/content/docs/partnerships.mdx:59]: making `'` for consistency
```suggestion
In an effort to support our self-serve model we've included links to resources, documentation, examples and best practices to guide you through the Vault integration development and testing process.
```

File: website/content/docs/partnerships.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+- Contributing to Nomad [guidelines](https://github.com/hashicorp/nomad/tree/main/contributing)
+- [Nomad Developer Community Forum](https://groups.google.com/g/nomad-tool)
+- [Nomad's source code](https://github.com/hashicorp/nomad)
+We encourage vendors to closely follow the above guidance. Adopting the same structure and coding patterns helps expedite the review and release cycles.
```
Comment (angrycub) [inline website/content/docs/partnerships.mdx:64]:
```suggestion
- [Nomad's source code](https://github.com/hashicorp/nomad)

We encourage vendors to closely follow the above guidance. Adopting the same structure and coding patterns helps expedite the review and release cycles.
```

File: website/content/docs/partnerships.mdx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+The review process can take a while to complete and may require some iterations through the code to address and problems identified by the HashiCorp team.
+
+### 5. Release 
+
+At this stage, it is expected that the integration is fully complete, the necessary documentation has been written, the acceptance tests have all passed, and that HashiCorp has reviewed the integration. Once the plugin has been validated and accepted by HashiCorp, the plugin can be hosted anywhere so it can easily be downloaded then installed within Nomad.
+
+Once the integration has been released the vendor is requested to sign the HashiCorp Technology Partner Agreement so that we can have their integration be listed on the HashiCorp website.
+ 
+### 6. Support
+Many vendors view the release step to be the end of the journey, while at HashiCorp we view it to be the beginning of the journey. Getting the integration built is just the first step in enabling users to leverage it against their infrastructure. Once development is completed, on-going effort is required to support the developed integration, maintain the provider and address any issues in a timely manner.
```
Comment (angrycub) [inline website/content/docs/partnerships.mdx:110]:
```suggestion
Many vendors view the release step to be the end of the journey, while at HashiCorp we view it to be the beginning of the journey. Getting the integration built is just the first step in enabling users to leverage it against their infrastructure. Once development is completed, on-going effort is required to support the developed integration, maintain the provider, and address any issues in a timely manner.
```

**Review by angrycub — APPROVED** [review]
Did a quick sweep.  Looks like there might be one typo and a few other nitpicks.  Should be mergeable after those.

**Pull request merged**